lambdas/service/app.py

```diff
@@ -86,7 +86,78 @@
 
 openapi_spec = {
     'info': {
-        'description': 'This should probably be really long',
+        'description': format_description('''
+            # Overview
+            Azul provides a programmatic interface for quickly and uniformly
+            querying metadata associated with experimental and analysis data
+            stored in the
+            [HCA Data Store (DSS)](https://github.com/HumanCellAtlas/data-store).
+            Azul serves as the backend for the
+            [HCA Data Browser](https://github.com/DataBiosphere/data-browser).
+
+            Azul denormalizes and aggregates metadata into several different indices:
```

Comment:
```suggestion
            Azul denormalizes, aggregates, and makes the metadata available through several different repository endpoints:
```
"Indices" is an Elasticsearch term we might want to leave out of this documentation? Also, not all indices have an associated repository endpoint (eg. projects & cell_suspension).